packages/core/src/components/forms/_controls.scss

```diff
@@ -34,10 +34,21 @@ $control-indicator-spacing: $pt-grid-size !default;
     background-color: $control-checked-background-color;
     box-shadow: $control-checked-box-shadow;
     color: $white;
+
+    @media (forced-colors: active) and (prefers-color-scheme: dark) {
+      background-color: $pt-high-contrast-mode-active-background-color;
+      // Windows High Contrast dark theme
+      border: 1px solid $pt-high-contrast-mode-active-background-color;
+    }
   }
 
   &:hover input#{$selector} ~ .#{$ns}-control-indicator {
     background-color: $control-checked-background-color-hover;
+
+    @media (forced-colors: active) and (prefers-color-scheme: dark) {
+      // Windows High Contrast dark theme
+      background-color: $pt-high-contrast-mode-active-background-color;
```

Comment:
not exactly related to this line of code, but I noticed we seem to be getting a nice background color for disabled & active radio indicators:

<img width="576" alt="Screen Shot 2022-09-26 at 5 23 06 PM" src="https://user-images.githubusercontent.com/723999/192383510-08714a34-995e-4f26-a368-91e30ae75952.png">

but we don't get the same for checkboxes:

<img width="594" alt="Screen Shot 2022-09-26 at 5 23 20 PM" src="https://user-images.githubusercontent.com/723999/192383531-8c0cc3ed-97dd-4c3e-849c-42e0164c12d9.png">

not a huge priority, but there may be an easy fix for this?

Reply:
hm i can fix - in any case, the disabled color is wrong for both